packages/augur-ui/src/modules/trading/betslip.tsx

```diff
@@ -19,8 +19,11 @@ import Styles from 'modules/trading/betslip.styles';
 import { PrimaryButton } from 'modules/common/buttons';
 
 export const Betslip = () => {
-  const [minimized, setMinimized] = useState(true);
-  const { theme } = useAppStatusStore();
+  const {
+    theme,
+    betslipMinimized,
+    actions: { setBetslipMinimized },
+  } = useAppStatusStore();
```

Comment:
on theme switch, if we are switching to `SPORTS` or `BETTING` we should make sure to reset `betslipMinimized` to minimized. This way when they switch it's always expected to start Minimized on tablet/mobile.